server/models/server-service.js

```diff
@@ -115,4 +115,20 @@ module.exports = function(ServerService) {
     ServerService.app.serviceManager.getDeployment(this, ctx.req, ctx.res);
   }
   ServerService.prototype.getPack = getPack;
+
+  ServerService.prototype.setEnv =
```

Comment:
nit, can you switch the way the function is added to prototype to match above methods please